obf-generation/travel-gpx/README.md

```diff
@@ -0,0 +1,144 @@
+GPX=osmand:shield_fg OBF=shield_fg POI,MAP,STYLE
+GPX=osmand:shield_fg_2 OBF=shield_fg_2 POI,MAP,STYLE
+GPX=osmand:shield_text OBF=shield_text POI,MAP,STYLE
+GPX=osmand:shield_textcolor OBF=shield_textcolor POI,MAP,STYLE
+GPX=osmand:shield_waycolor OBF=shield_waycolor POI,SPECIAL (shadow copy of osmc_waycolor)
+```
+
+## 4. Save/Restore (analytics, elevation and special tags)
+
+```
+OBF=route_id POI,MAP,SPECIAL (used to identify unique GPX and to connect Map<->POI data)
+
+OBF=route_radius POI (specify default radius to search in POI section)
+
+OBF=distance POI (distance of all GPX segments)
+OBF=max_speed,avg_speed,min_speed POI (speed analytics)
+OBF=time_span,time_span_no_gaps,time_moving,time_moving_no_gaps POI (time analytics)
+OBF=start_ele,ele_graph MAP (GPX-specified elevation data wrapped in compact binary array)
+OBF=min_ele,avg_ele,max_ele,diff_ele_up,diff_ele_down POI,MAP (analytics of GPX-specified elevation data)
+```
+
+## 5. Rendering (Map-section)
+
+### 5.1. Tracks
+
+```
+GPX=<trkseg> OBF=route VAL=segment
+
+GPX=osmand:color OBF=color POI,MAP,STYLE,SPECIAL
+GPX=osmand:width OBF=gpx_width POI,MAP,STYLE (fixed width values supported)
+
+GPX=osmand:shield_bg OBF=shield_bg POI,MAP,STYLE
+GPX=osmand:shield_fg OBF=shield_fg POI,MAP,STYLE
+GPX=osmand:shield_fg_2 OBF=shield_fg_2 POI,MAP,STYLE
+GPX=osmand:shield_text OBF=shield_text POI,MAP,STYLE
+GPX=osmand:shield_textcolor OBF=shield_textcolor POI,MAP,STYLE
+
+GPX=osmand:use_osmc_colors OBF=use_osmc_colors POI,MAP,STYLE (transparent colors)
+```
+
+### 5.2. Waypoints
+
+```
+GPX=<wpt> OBF=route VAL=point
+GPX=osmand:icon OBF=gpx_icon POI,STYLE
+GPX=osmand:color OBF=color POI,MAP,STYLE,SPECIAL
+GPX=osmand:background OBF=gpx_bg POI,STYLE (mix color+gpx_bg is badly supported due to lack of icons)
+```
+
+## 6. Categories/Search (POI-section)
+
+### 6.1. POI-categories (search, filters, show-on-map)
+
+```
+GPX=osmand:osm_tag_route OBF=route_type VALUE=cycling POI (Bicycle)
+GPX=osmand:osm_tag_route OBF=route_type VALUE=fitness POI (Fitness trails)
+GPX=osmand:osm_tag_route OBF=route_type VALUE=hiking POI (Hiking)
+GPX=osmand:osm_tag_route OBF=route_type VALUE=inline_skates POI (Inline skates)
+GPX=osmand:osm_tag_route OBF=route_type VALUE=mountainbike POI (Mountain biking)
+GPX=osmand:osm_tag_route OBF=route_type VALUE=riding POI (Horse riding)
+GPX=osmand:osm_tag_route OBF=route_type VALUE=running POI (Running)
+GPX=osmand:osm_tag_route OBF=route_type VALUE=snowmobile POI (Snowmobile)
+GPX=osmand:osm_tag_route OBF=route_type VALUE=walking POI (Walking)
+GPX=osmand:osm_tag_route OBF=route_type VALUE=water POI (Water sports)
+GPX=osmand:osm_tag_route OBF=route_type VALUE=winter POI (Winter sports)
+
+GPX=[default] OBF=route_type VALUE=track POI (Other routes)
+
+GPX=<wpt> OBF=route_track_point POI (Other routes points)
+```
+
+### 6.2. Search (by ref/name)
+
+```
+GPX=<name>|osmand:osm_tag_name|osmand:osm_tag_ref|osmand:osm_tag_description OBF=name POI,SPECIAL
```

Comment:
Should we use osm_tag_name, osm_tag_ref? 

1. Search  by osmand:activity and by https://www.topografix.com/gpx/1/1/#type_metadataType keywords (can leave for later if not implemented)
2. Name = from any existing (order): trk.extensions.osmand:name, trk.name, metadata.extensions:osmand:name,
metadata.name, 
3. Ref = from any existing (order preserved): trk.extensions.osmand:ref,metadata.extensions:osmand:ref